### ISSUE TITLE
Can't drag ground-targeted abilities on the actionbar

### ISSUE THREAD

**Issue by dualcoding** [cogwerkz/DiabolicUI#54]
**Description**
Dragging icons in the actionbar with alt-ctrl-shift activates them. When trying to move ground-targeted abilities the reticle shows up and the icon can't be moved (workaround: swap with another spell first)

**Version Number** 1.2.43-release

**Steps to Reproduce**
1. Try to drag a ground-targeted ability from the actionbar holding alt-ctrl-shift.

**Expected Behavior**
Spell icon should be picked up without activating.

**Actual Behavior**
Spell activates, and in the case of ground-targeted abilities, icon is not picked up.

**Only DiabolicUI Enabled** Yes

**Additional Information**
Keep up the great work, have already switched to using full-time.

**Comment by dualcoding** [comment]
I'm guessing this is because DiabolicUI activates on click down compared to default UI click up. Not a bad thing!

**Comment by goldpaw** [comment]
Actually DiabolicUI follows the blizzard settings for whether it should cast on down or when releasing the button. But Blizzard removed the setting from their menu in Legion, yet kept the setting in the game. Chaotic, eh? ;) 

I'll add that setting to my actionbar toggle menu, since it's been annoying me too. I'll be changing this to a feature request issue! 👍 

For now, you can change it by either installing the addon AdvancedIntefaceOptions from Curse, or by typing the following two lines each followed by the enter key in the chat. Make sure your character isn't engaged in combat when doing it, btw, because then it'll give you an error! 

/run SetCVar("ActionButtonUseKeyDown", 0)
/reload

**Comment by dualcoding** [comment]
Figures! I actually did change that CVar, but forgot to reload. Then disabled Diabolic and hey, it worked differently in the blizzard UI (after reload). User error ;)

But still, a toggle would be appreciated!

**Comment by goldpaw** [comment]
Yeah, currently DiabolicUI reads the Blizzard value after a reload or login, but doesn't update it on-the-fly if the CVar is changed from somewhere else. Will fix that too. :)

**Comment by Anderzenn** [comment]
I was wondering if the default blizzard ui maybe checks if the player is holding shift? And if so they do so you can't use the spell while holding shift? Just a thought I had 😄

**Comment by goldpaw** [comment]
Any mousekey and any modifiers we hold down are sent to the actionbutton in question. Yes, the UI reacts to SHIFT, ALT and CTRL, but no; unless you specifically has created a macro at that button that avoids SHIFT on purpose, the spell will still be cast.
